src/index.js

```diff
@@ -1,9 +1,8 @@
-import Formio from './formio';
-global.Formio = Formio;
-export {default as FormioForm} from './formio.form';
-export {default as FormioWizard} from './formio.wizard';
-export {default as FormioPDF} from './formio.pdf';
+export {default as Formio} from './Formio';
+export {default as Form} from './Form';
+export {default as Wizard} from './Wizard';
+export {default as PDF} from './PDF';
 export {default as createForm} from './createForm';
 export {default as formFactory} from './formFactory';
-export {Formio};
-export default Formio;
+export {default as FormBuilder} from './Builder';
```

Comment:
We should keep this name consistent. That way the imports would be
import {FormBuilder} from 'formiojs';
or
import FormBuilder form 'formiojs/FormBuilder';

or just use Builder for both.

Reply:
Ok